web/src/components/Collapsible.js

```diff
@@ -1,5 +1,4 @@
 import deline from 'deline';
-import kebabCase from 'lodash.kebabcase';
```

Comment:
While I was tracing the route of the Activity components, I ran across this. All it seems to do is replace spaces/special chars with dashes and lowercase things, both of which are easy to do with regex in vanilla JS. Seemed lighter weight than importing something that's only being used once in the file (especially since it only needs to care about two special chars. 🤷‍♀️